modules/tinymce/src/core/main/ts/api/dom/Selection.ts

```diff
@@ -542,11 +544,21 @@ const EditorSelection = (dom: DOMUtils, win: Window, serializer: DomSerializer,
     controlSelection.destroy();
   };
 
+  /**
+   * Expands the range of the selection to contain the entire word when the selection is collapsed within the word
```

Comment:
```suggestion
   * Expands the selection range to contain the entire word when the selection is collapsed within the word.
```

Copy edit suggestion: active voice centres what is expanded.